# patch to adjust paths in runall script according to our GENTOO
# install

--- gamess/runall	2005-07-05 14:08:04.000000000 +0000
+++ gamess-gentoo/runall	2005-09-05 11:05:31.000000000 +0000
@@ -10,7 +10,7 @@
 #  Edit this file to set the base path to GAMESS, and execute it to be
 #  sure all the jobs TERMINATED NORMALLY, and got correct numbers too.
 #
-chdir /u1/mike/gamess
+#chdir /u1/mike/gamess/
 set VERNO=00
 #
 foreach NUM ( 01 02 03 04 05 06 07 08 09 10 11 12 13 14 15 \
